packages/ember-metal/lib/property_get.js

```diff
@@ -62,7 +62,7 @@ export function get(obj, keyName) {
   Ember.assert(`Cannot call get with ${keyName} key.`, !!keyName);
   Ember.assert(`Cannot call get with '${keyName}' on an undefined object.`, obj !== undefined);
 
-  if (!obj) {
+  if (!obj && typeof obj !== 'string') {
```

Comment:
I think an `isNone` (via `import isNone from 'ember-metal/is_none';`) check here would be better  than specifically checking for strings.